src/frontend/src/btc/components/send/BtcSendReview.svelte

```diff
@@ -21,15 +25,30 @@
 	export let source: string;
 	export let utxosFee: UtxosFee | undefined = undefined;
 
+	const { sendBalance, sendTokenDecimals } = getContext<SendContext>(SEND_CONTEXT_KEY);
+
 	let hasPendingTransactionsStore: Readable<BtcPendingSentTransactionsStatus>;
 	$: hasPendingTransactionsStore = initPendingSentTransactionsStatus(source);
 
+	// TODO: in the updated Send flow designs, this check will be done one step before in BtcSendForm
+	let insufficientFundsForFee: boolean;
+	$: insufficientFundsForFee =
+		nonNullish(utxosFee) && nonNullish($sendBalance) && nonNullish(amount)
+			? parseToken({
+					value: `${amount}`,
+					unitName: $sendTokenDecimals
+				})
+					.add(utxosFee.feeSatoshis)
+					.gt($sendBalance)
+			: false;
+
 	let disableSend: boolean;
 	// We want to disable send if pending transactions or UTXOs fee isn't available yet, there was an error or there are pending transactions.
 	$: disableSend =
```

Comment:
Unrelated: In a follow up PR, should we extract this logic in some function? It is becoming quite complex. WDY?

Reply:
Yes, makes sense!